OWNERS_ALIASES

```diff
@@ -2,31 +2,74 @@
 
 aliases:
   sig-cluster-lifecycle-leads:
-    - luxas
-    - justinsb
-    - timothysc
+  - luxas
```

Comment:
s/luxas/neolit123/

Lukas is now an emeritus chair and Lubomir was promoted. Might be good to add fabrizio here as well since he is now a Tech lead for SIG Cluster lifecycle. https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle#chairs